app/views/visits/show.html.haml

```diff
@@ -34,6 +34,15 @@
       %br
 
 %h3= t('.diagnosis')
-= link_to visit_diagnosis_index_path(visit_id: @visit.id), class: 'ui medium green button' do
+%table.ui.celled.table.ui.selectable
+  %tbody
+    - @visit.diagnoses.each do |diagnosis|
+      %tr
+        %td
+          = link_to visit_diagnosis_path(visit_id: diagnosis.visit_id, id: diagnosis.id) do
+            .full-width
+              = t('.diagnosis_date', date: I18n.l(diagnosis.created_at.to_date))
```

Comment:
Add a method in diagnosis model such as :
```
def created_at_date_localized
  I18n.l(created_at.to_date)
end
```
And call it like this `diagnosis.created_at_date_localized`

(Or name it `creation_date_localized`)